packages/platform/libraries/index.ts

```diff
@@ -82,7 +82,7 @@ export { getAvailableSlots };
 export type AvailableSlotsType = Awaited<ReturnType<typeof getAvailableSlots>>;
 export { handleNewRecurringBooking } from "@calcom/features/bookings/lib/handleNewRecurringBooking";
 
-export { getConnectedDestinationCalendars } from "@calcom/lib/getConnectedDestinationCalendars";
+export { getConnectedDestinationCalendarsAndEnsureDefaultsInDb } from "@calcom/lib/getConnectedDestinationCalendars";
```

Comment:
I don't see this function being used in api v2

Reply:
whenever platform libraries is changed it needs to be published on npm too

Reply:
I realized this and then exposed the older name as well.  I can use the same name here as well. Using he same in calendars.service.ts